crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -118,18 +121,68 @@ impl Plugin for VisibilityPlugin {
     }
 }
 
+/// Calculates and updates [`Aabb`]s for [`Entities`](Entity) with [`Mesh`]es.
+/// To opt out of bound calculation for an `Entity`, give it the [`NoFrustumCulling`] component.
```

Comment:
Keep these ^ two lines of documentation of the system, but I don't feel the rest is useful personally.

Reply:
Maintained in 9730917d7a5ea6082b8966f5f6e3dd12724d7317.